### PR TITLE
ZH-trans: add update-api-object-kubectl-patch.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+-->
+
+## 使用策略合并补丁更新部署
+
+<!--
+Here's the configuration file for a Deployment that has two replicas. Each replica
+is a Pod that has one container:
+-->
+
+下面是具有两个副本的部署的配置文件。每个副本是一个 Pod，有一个容器：
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: Deployment 不要翻译

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+
+{{< codenew file="application/deployment-patch.yaml" >}}
+
+<!--
+Create the Deployment:
+-->
+
+创建部署：
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: 同上

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+
+```shell
+kubectl create -f https://k8s.io/examples/application/deployment-patch.yaml
+```
+
+<!--
+View the Pods associated with your Deployment:
+-->
+查看与部署相关的 Pod：
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: 同上

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+
+```shell
+kubectl get pods
+```
+
+<!--
+The output shows that the Deployment has two Pods. The `1/1` indicates that
+each Pod has one container:
+-->
+输出显示部署有两个 Pod。`1/1` 表示每个 Pod 有一个容器:
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: 同上

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 5]
```diff
@@ -0,0 +1,507 @@
...
+NAME                        READY     STATUS    RESTARTS   AGE
+patch-demo-28633765-670qr   1/1       Running   0          23s
+patch-demo-28633765-j5qs3   1/1       Running   0          23s
+```
+
+<!--
+Make a note of the names of the running Pods. Later, you will see that these Pods
+get terminated and replaced by new ones.
+-->
+把运行的名字记下来。稍后，您将看到这些 Pod 被终止并被新的 Pod 替换。
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: 运行的 Pods 的名字

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 6]
```diff
@@ -0,0 +1,507 @@
...
+    spec:
+      containers:
+      - name: patch-demo-ctr-2
+        image: redis
+```
+
+<!--
+Patch your Deployment:
+-->
+补丁部署：
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: 修补您的 Deployment

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 7]
```diff
@@ -0,0 +1,507 @@
...
+
+```shell
+kubectl patch deployment patch-demo --patch "$(cat patch-file-containers.yaml)"
+```
+<!--
+View the patched Deployment:
+-->
+查看补丁部署：
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: 查看修补后的 Deployment

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 8]
```diff
@@ -0,0 +1,507 @@
...
+
+```shell
+kubectl get deployment patch-demo --output yaml
+```
+
+<!--
+The output shows that the PodSpec in the Deployment has two Containers:
+-->
+输出显示部署中的 PodSpec 有两个容器:
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: Deployment

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 9]
```diff
@@ -0,0 +1,507 @@
...
+- image: nginx
+  imagePullPolicy: Always
+  name: patch-demo-ctr
+  ...
+```
+
+<!--
+View the Pods associated with your patched Deployment:
+-->
+查看与补丁部署相关的 Pod:
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: Deployment

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 10]
```diff
@@ -0,0 +1,507 @@
...
+kubectl get pods
+```
+
+<!--
+The output shows that the running Pods have different names from the Pods that
+were running previously. The Deployment terminated the old Pods and created two
+new Pods that comply with the updated Deployment spec. The `2/2` indicates that
+each Pod has two Containers:
+-->
+输出显示正在运行的 Pod 与以前运行的 Pod 有不同的名称。部署终止了旧的 Pod，并创建了两个
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: Deployment

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 11]
```diff
@@ -0,0 +1,507 @@
...
+```
+NAME                          READY     STATUS    RESTARTS   AGE
+patch-demo-1081991389-2wrn5   2/2       Running   0          1m
+patch-demo-1081991389-jmg7b   2/2       Running   0          1m
+```
+
+<!--
+Take a closer look at one of the patch-demo Pods:
+-->
+仔细查看其中一个补丁-演示 Pod:
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: patch-demo 不要翻译

File: content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md [hunk 12]
```diff
@@ -0,0 +1,507 @@
...
+  ...
+- image: nginx
+  ...
+```
+
+<!--
+### Notes on the strategic merge patch
+-->
+
+### 战略合并补丁的说明
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/update-api-object-kubectl-patch.md]: strategic merge patch：策略性合并类的修补

**Comment by xichengliudui** [comment]
已修改。

**Comment by xichengliudui** [comment]
已修改。

**Pull request merged**